server/storeProfile.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+require_once 'config.php';
+
+storeProfile(_$POST);
+
+function storeProfile($data) {
+  GLOBAL $DB_CONN;
+  $response = array();
+  $response['success'] = false;
+
+  $DB_CONN -> beginTransaction();
+
+  try {
+    $clientInsert = "INSERT INTO vita.user
+    (
+      firstName,
+      lastName,
+      phone,
```

Comment:
Also, did you look at the ERD for the User table? There is no column called "phone". https://github.com/UNL-CSE-AMBASSADORS/VITA/pull/96